Makefile

```diff
@@ -1,5 +1,5 @@
 all:
 	bash -c "source venv/bin/activate;\
-	tox -e py36,coverage,mypy;\
+	tox -e coverage,mypy;\
```

Comment:
please re-add py36

Reply:
coverage already runs pytest. That means when running make, all tests are ran twice

Reply:
OK that's fine for the makefile then